pkg/command/command.go

```diff
@@ -194,7 +193,14 @@ func (c *Command) run(printOutput bool) (res *Status, err error) {
 		// Wait for last command in the pipe to finish
 		if i+1 == len(c.cmds) {
 			runErr = cmd.Wait()
```

Comment:
@saschagrunert I believe the test is failing because `cmd.Wait()` is closing the pipes before we finish reading from `stdout` and `stderr`

Reply:
Yes I agree. I can try to turn it around with the implementation below, but I thought that my error check for `os.ErrClosed` is already _a solution_. :grin: I'll give it a try.